modules/candidate_list/README.md

```diff
@@ -0,0 +1,51 @@
+# Candidate List
+
+## Purpose
+
+The candidate list is intended to provide a menu from which users
+can access specific candidates while browsing LORIS data.
```

Comment:
The most important piece of context is how this module is used/known:  
e.g. I'd add:
"This module is known to users in the front end as "Access Profiles" and is the most heavily accessed module in LORIS.   

It provides an at-a-glance view of data available across all candidates stored in LORIS."

Reply:
`..is the most heavily accessed module in LORIS.` seems speculative

Reply:
I think the note about the front-end name is a good suggestion, though.

Reply:
I added something about the frontend.